### PR TITLE
added AC for last updated date

### PR DESCRIPTION
**Pull request by talawson05**
Clarified the change of wording from last changed date to last updated date with @JoshuaRobertsBJSS

### CODE REVIEW

**Comment by talawson05** [comment]
Cancelling this, I think this might be in the wrong project

File: src/MarketingPageAcceptanceTestsSpecflow/Steps/AboutSolution/LastUpdatedDate.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+using MarketingPageAcceptanceTestsSpecflow.Utils;
+using TechTalk.SpecFlow;
+
+namespace MarketingPageAcceptanceTestsSpecflow.Steps.AboutSolution
+{
+    [Binding]
+    public class LastUpdatedDate
+    {
+        private UITest _test;
+        private ScenarioContext _context;
```
Comment (PhysicsJon) [inline src/MarketingPageAcceptanceTestsSpecflow/Steps/AboutSolution/LastUpdatedDate.cs]: I think both these fields should have `readonly` applied, but that applies across the solution, so I might get those done in another PR

File: src/MarketingPageAcceptanceTestsSpecflow/Steps/AboutSolution/LastUpdatedDate.cs [hunk 2]
```diff
@@ -39,8 +39,14 @@ public void WhenTheContentHasBeenUpdated()
...
-        [Then(@"the Last Changed Date is updated")]
-        public void ThenTheLastChangedDateIsUpdated()
+        [Then(@"the Last Changed Date is updated in the SolutionDetail table")]
+        public void ThenTheLastChangedDateIsUpdatedInSolutionDetail()
+        {
+            _context.Pending();
+        }
+
+        [Then(@"the Last Changed Date is updated in the MarketingContact table")]
+        public void ThenTheLastChangedDateIsUpdatedInMarketingContact()
```
Comment (PhysicsJon) [inline src/MarketingPageAcceptanceTestsSpecflow/Steps/AboutSolution/LastUpdatedDate.cs]: Can these 2 methods be parameterised?

Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTestsSpecflow/Steps/AboutSolution/LastUpdatedDate.cs]:
```
[Then(@"the Last Changed Date is updated in the (.*) table")]
public void ThenTheLastChangedDateIsUpdated(string tableName)
```

Comment (talawson05) [reply src/MarketingPageAcceptanceTestsSpecflow/Steps/AboutSolution/LastUpdatedDate.cs]: totally, would you prefer i did this now or during implementation?

Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTestsSpecflow/Steps/AboutSolution/LastUpdatedDate.cs]: Prefer now, then we can estimate better how long it's going to take and write the SQL correctly from the off

**Pull request merged**